README.md

```diff
@@ -65,3 +65,29 @@ You can set the file extension used by the UI to download a result from your fun
 * `environment`
 
 Set timeouts etc
+
+# OpenFaaS Template Store
+
+The Template Store are all the templates from OpenFaaS and OpenFaaS-Incubator organizations, along with user-defined templates which were approved by the community.
```

Comment:
'The Template store _contains_ ....'
'which _have been_ approved by the community.'
Can we link OpenFaaS-Incubator to https://github.com/openfaas-incubator, please?

Reply:
Done ✔️